lib/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Util = require('util');
```

Comment:
Nit: Generally we separate external and internal file imports with a newline, as well as alphabetize them and align their `=`'s. See [this section](https://github.com/lob/lob-api/blob/master/src/plugins/features/addresses/index.js#L3-L10) from `lob-api` as an example.